video_stream.py

```diff
@@ -5,11 +5,10 @@
 import typing
 
 import numpy
-from picamera.array import PiRGBArray
-from picamera import PiCamera
+from picamera2 import Picamera2
 
 import logger
-
+import logging
```

Comment:
`logger` deliberately exposes log levels so you don't have to import `logging` again